src/BlockTypes/Cart.php

```diff
@@ -103,6 +100,27 @@ protected function enqueue_data( array $attributes = [] ) {
 		do_action( 'woocommerce_blocks_cart_enqueue_data' );
 	}
 
+	/**
+	 * Removes accents from an array of values, sorts by the values, then returns the original array values sorted.
+	 *
+	 * @param array $array Array of values to sort.
+	 * @return array Sorted array.
+	 */
+	protected function deep_sort_with_accents( $array ) {
```

Comment:
This could become a utility but there is no obvious home for it so let's keep it across both files for this iteration and move if it becomes useful in other contexts.